### PR TITLE
Upgrade to babel 7

### PR DESCRIPTION
**Pull request by TrySound**
Linaria will require a lot of changes so I decided to use different
solution to not rewrite styles.

After merging this I will upgrade each plugin with linaria individually.

### CODE REVIEW

File: draft-js-mention-plugin/package.json [hunk 1]
```diff
@@ -27,18 +27,17 @@
...
+    "build:css": "node ../scripts/build-css.js $(pwd)",
+    "prepublish": "yarn build"
   },
   "license": "MIT",
   "dependencies": {
     "clsx": "^1.0.4",
     "immutable": "~3.7.4",
     "lodash": "^4.17.14",
-    "prop-types": "^15.5.8",
-    "union-class-names": "^1.0.0"
```
[the remark this answers is not in the document]
Comment (TrySound) [reply draft-js-mention-plugin/package.json]: Oops, looks like we missed it before :)

File: draft-js-plugins-editor/src/index.js [hunk 2]
```diff
@@ -2,6 +2,6 @@ import createEditorStateWithTextFn from './utils/createEditorStateWithText';
 import composeDecoratorsFn from './utils/composeDecorators';
 
 // eslint-disable-next-line import/no-named-as-default
-export default from './Editor';
+export { default } from './Editor';
```
[the remark this answers is not in the document]
Comment (TrySound) [reply draft-js-plugins-editor/src/index.js]: This is not standard syntax.
```
export default from './Editor';
```
This is the same but does not require babel plugin
```
export { default } from './Editor';
```

File: package.json [hunk 3]
```diff
@@ -75,15 +75,22 @@
     "trailingComma": "es5"
   },
   "devDependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.5.5",
```
[the remark this answers is not in the document]
Comment (TrySound) [reply package.json]: I'm afraid to break storybook :)

File: scripts/build-css.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+const { sync: rimraf } = require('rimraf');
+
+const root = process.argv[2];
+
+let content = '';
+matched('lib-css/**/*.css', { cwd: root }).forEach(file => {
+  content += fs.readFileSync(file, 'utf-8');
+});
+fs.writeFileSync(path.join(root, 'lib', 'plugin.css'), content);
+rimraf(path.join(root, 'lib-css'));
```
[the remark this answers is not in the document]
Comment (TrySound) [reply scripts/build-css.js]: Nope, linaria will require this too. Same as with babel-plugin-webpack-loaders

File: draft-js-mention-plugin/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (TrySound) [reply draft-js-mention-plugin/package.json]: Not needed, right.

File: draft-js-plugins-editor/src/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (juliankrispel) [reply draft-js-plugins-editor/src/index.js]: oh ok

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (juliankrispel) [reply package.json]: oh ok

File: scripts/build-css.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (juliankrispel) [reply scripts/build-css.js]: ok

**Pull request merged**